src/dso_api/dynamic_api/filters/values.py

```diff
@@ -0,0 +1,136 @@
+"""Conversion of query-string values to Python types."""
+from __future__ import annotations
+
+import math
+import re
+from datetime import date, datetime, time
+from decimal import Decimal
```

Comment:
Why are we using `decimal` here?

Reply:
JavaScript number has a higher precision then a Python float.

Reply:
According to the [ECMAScript spec](https://262.ecma-international.org/11.0/), JavaScript numbers are IEEE 754 double precision (64 bits). Python doesn't guarantee anything, but CPython uses C's `double` type, which is also IEEE 754 double precision, on all major platforms.

Reply:
chosen to leave as it is